readme.md

```diff
@@ -386,6 +386,7 @@ If we don't support your language, please consider contributing! You can find mo
 #### Repo Card Exclusive Options
 
 *   `show_owner` - Shows the repo's owner name *(boolean)*. Default: `false`.
+*   `description_lines_count` - Manually set the number of lines for the description *(number)*. If this parameter is not specified, the number of lines will be automatically adjusted according to the length of the description.
```

Comment:
Maybe we can state that the maximum is 3.

Reply:
Further, it looks like when the description is longer than the set `description_lines_count`, the maximum is used. You can check this out by looking at https://github.com/doersino/tweakpane-plugin-infodump. Here, we can do two things, or we use an ellipsis and truncate the text or document the behavoir.